neos/neos-development-collection
# BUGFIX: re-add lost variants update functionality

Re-add lost variants update functionality to fix issue #2884 

**What I did**
I compared the changes of  PR #2751 with the code befor the changes of this PR and saw, that the line `$this->getRepository($variant)->update($variant);` was missing after the changes of the PR. I added the missing line where I think it is necessary. Additionally i added the  generation of the redirect to the else part in the replacement function, which was also missing. I think this fixes issue #2884.

**How to verify it**
Find an asset, for which the resource can not be replaced, for example on the demo site. Then  checkout my changes and try replacing the resource of the found asset. The replcement should be possible again.
